### PR TITLE
Eclipse Code Change Restructure

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

### CODE REVIEW

**Comment by j-c-berger** [comment]
For Pr 1757: https://github.com/eclipse/codewind/issues/1757

@micgibso and/or @sishida, please review and assign me, thanks.

File: docs/_documentations/mdt-eclipse-codechange.md [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
+# Making a code change with Eclipse 
 <br/>
-Codewind will automatically build and redeploy your application whenever you make a code change and save the file.
+Codewind automatically builds and redeploys your application whenever you make a code change and save the file.
 
-To see this in action, we can make a change to the getting started example.
+To see this feature in action, make a change to the getting started example.
 
-First step is to edit the index.html file
+1\. Edit the index.html file.
```
Comment (sishida) [inline docs/_documentations/mdt-eclipse-codechange.md]: Code phrase format is good for file names: `index.html`

File: docs/_documentations/mdt-eclipse-codechange.md [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
 
 ![](dist/images/eclipsechangeproject1.png){:width="800px"}
 
-and navigate to the bottom of the file to find the lines
+2\. Navigate to the bottom of the file to find the lines.
```
Comment (sishida) [inline docs/_documentations/mdt-eclipse-codechange.md]: Here's what Acrolinx says about the word "Navigate": `Use only when a more direct verb, such as "click," "select," "browse," or "go to," is not applicable. Use only to provide direction to a location (for example, "Navigate to the directory that contains the files that you want to edit").` In this case, "go" works great. :)

**Comment by j-c-berger** [comment]
@sishida, applied your feedback in the latest commit :)

**Pull request merged**